site/content/contribute/webapp/developer-setup.md

```diff
@@ -5,24 +5,35 @@ weight: 2
 subsection: Web App
 ---
 
-# Developer Setup
+Set up your development environment for building, running, and testing the Mattermost web app.
 
-## Prerequisites
-- Git
-- homebrew (for Mac)
+#### Prerequisites
 
-## Setup
+1. Set up your [development environment for the Mattermost server](/contribute/server/developer-setup). 
 
-Follow these steps to set up your web app developer environment:
+ - Running these steps will also install Git on your machine, and Homebrew for Mac.
 
-1. To develop for the web app you're also going to need to [set up the developer environment for the server](/contribute/server/developer-setup). So if you haven't done that yet, start there and then come back here.
+2. Install Node.js and npm following the instructions at https://www.npmjs.com/get-npm.
 
-2. Install [Node.js and npm](https://www.npmjs.com/get-npm).
+3. On Mac, install libpng:
 
-3. If you're on Mac, you'll need to install libpng as well: `brew install libpng`
+    ```sh
+    brew install libpng
+    ```
 
-4. Go to https://github.com/mattermost/mattermost-webapp and fork the repository.
+#### Setup
 
-5. Clone your fork `git clone https://github.com/<yourgithubusername>/mattermost-webapp` beside your `mattermost-server`.
+1. Go to https://github.com/mattermost/mattermost-webapp and create a fork.
```

Comment:
The location of the fork is important: various scripts currently assume it to be a sibling to the `mattermost-server` directory. Also, there's been some recent confusion about a missing symlink. Suggest:

1. Go to https://github.com/mattermost/mattermost-webapp and create a fork.
2. Clone the fork next to your mattermost-server directory, replacing `{yourgithubusername}` with your GitHub username:
```sh
git clone https://github.com/{yourgithubusername}/mattermost-webapp.git $(go env GOPATH)/go/src/mattermost/mattermost-webapp
```
3. Link your server with the webapp:
```
ln -s $(go env GOPATH)/src/github.com/mattermost/mattermost-webapp/dist $(go env GOPATH)/src/github.com/mattermost/mattermost-server/client
```
4. Test your environment:
```sh
cd $(go env GOPATH)/go/src/mattermost/mattermost-webapp
make test
```

Reply:
why not using `go get`?

Reply:
I could be mistaken, but I don't think `go get` works with non-Go repositories? We'll eventually adopt Go modules, at which point `go get` isn't the right path forward, since modules means pivoting away from `$GOPATH`.

Reply:
It works: I tested it. The only problem is that it trigger a warning and at the beggining I thought it failed.

Why mattermost-server isn't a Go repository?